data/campaigns/Winds_of_Fate/scenarios/01_The_Hunt.cfg

```diff
@@ -58,18 +58,18 @@
     [/side]
     {WOF_DEATHS}
 
-    {SPAWNER_SIDE 2  "Stymphalian"    Mm      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}
-    {SPAWNER_SIDE 3  "Dragonfly"      Ss      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}
-    {SPAWNER_SIDE 4  "Leopard"        *^Ftr   {ON_DIFFICULTY4 0.1  0.25  0.5  1}   6 1}
-    {SPAWNER_SIDE 5  "Horned Scarab"  *^Ft    {ON_DIFFICULTY4 0.1  0.25  0.5  1}   6 1}
-    {SPAWNER_SIDE 6  "Fire Ant"       Dd      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}
-    {SPAWNER_SIDE 7  "Rock Scorpion"  Mm      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   1 1}
-    {SPAWNER_SIDE 8  "Swamp Lizard"   Ss      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   6 1}
-    {SPAWNER_SIDE 9  "Water Serpent"  Wwt^Ew* {ON_DIFFICULTY4 0.05 0.125 0.25 0.5} 3 1}
-    {SPAWNER_SIDE 10 "Dolphin"        Ww      0.05                                 3 2}
-    {SPAWNER_SIDE 11 "Nibbler"        Wwt     {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}
-    {SPAWNER_SIDE 12 "Cuttle Fish"    Wo*     {ON_DIFFICULTY4 0.05 0.125 0.25 0.5} 6 2}
-    {SPAWNER_SIDE 13 "Great Seahorse" Wwr     0.05                                 3 2}
+    {SPAWNER_SIDE 2  "Stymphalian"    Mm      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}    # wmllint: skip-side
+    {SPAWNER_SIDE 3  "Dragonfly"      Ss      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}    # wmllint: skip-side
+    {SPAWNER_SIDE 4  "Leopard"        *^Ftr   {ON_DIFFICULTY4 0.1  0.25  0.5  1}   6 1}    # wmllint: skip-side
+    {SPAWNER_SIDE 5  "Horned Scarab"  *^Ft    {ON_DIFFICULTY4 0.1  0.25  0.5  1}   6 1}    # wmllint: skip-side
+    {SPAWNER_SIDE 6  "Fire Ant"       Dd      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}    # wmllint: skip-side
+    {SPAWNER_SIDE 7  "Rock Scorpion"  Mm      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   1 1}    # wmllint: skip-side
+    {SPAWNER_SIDE 8  "Swamp Lizard"   Ss      {ON_DIFFICULTY4 0.1  0.25  0.5  1}   6 1}    # wmllint: skip-side
+    {SPAWNER_SIDE 9  "Water Serpent"  Wwt^Ew* {ON_DIFFICULTY4 0.05 0.125 0.25 0.5} 3 1}    # wmllint: skip-side
+    {SPAWNER_SIDE 10 "Dolphin"        Ww      0.05                                 3 2}    # wmllint: skip-side
+    {SPAWNER_SIDE 11 "Nibbler"        Wwt     {ON_DIFFICULTY4 0.1  0.25  0.5  1}   9 2}    # wmllint: skip-side
+    {SPAWNER_SIDE 12 "Cuttle Fish"    Wo*     {ON_DIFFICULTY4 0.05 0.125 0.25 0.5} 6 2}    # wmllint: skip-side
+    {SPAWNER_SIDE 13 "Great Seahorse" Wwr     0.05                                 3 2}    # wmllint: skip-side
```

Comment:
The `skip-side` directives are only needed if there's another side defined afterwards without using a macro. Only S05 needs them, because of the Wisp side.

Reply:
It still doesn't hurt to put them in anyway though. That's what I said in the other thread too, which I think is why he did it.

Reply:
Just let me know which way seems like best coding practice - to remove them as unnecessary clutter or to keep them as future proofing.

Reply:
I guess they might as well go in. My feelings are based on the name of the wmllint pragma - if it had been called `count-side` instead of `skip-side` then I'd be pushing for them to be included, which means I'm not forming a technically sound opinion on this one.